### PR TITLE
update yougthous config, bump version (0.4.0-rc1), rename config keys to `:xiana/...`

### PR DESCRIPTION
**Pull request by jehaby**
update yougthous config, bump version (0.4.0-rc1), rename config keys to `:xiana/...`

more info on zulip

### CODE REVIEW

**Review by g-krisztian — COMMENTED** [review]
Looks good, pls update framework's version number too

**Review by g-krisztian — APPROVED** [review]

File: examples/cli-chat/docker-compose.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ services:
     build:
       context: ./Docker
       dockerfile: db.Dockerfile
+    environment:
+      - POSTGRES_PASSWORD=dev
```
Comment (soulflyer) [inline examples/cli-chat/docker-compose.yml:8]: and here

File: examples/controllers/docker-compose.yml [hunk 2]
```diff
@@ -6,6 +6,8 @@ services:
       dockerfile: db.Dockerfile
     volumes:
       - db-data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_PASSWORD=dev
```
Comment (soulflyer) [inline examples/controllers/docker-compose.yml:10]: and here

**Review by soulflyer — CHANGES_REQUESTED** [review]
Looks good, just needs the passwords to match so the examples work as per their read-me files

**Pull request merged**